app/jobs/replicated_file_check_job.rb

```diff
@@ -10,13 +10,24 @@ class ReplicatedFileCheckJob < ApplicationJob
 
   # @param [PreservedCopy] verify that the archived preserved_copy exists on an endpoint
   def perform(preserved_copy)
-    return if preserved_copy.unreplicated? # FIXME: storytime (Monday 07/02)
+    if preserved_copy.unreplicated?
+      Rails.logger.error("Archival Preserved Copy: #{preserved_copy} has a status of #{preserved_copy.status}.")
+      return
+    end
     aws_s3_object = bucket.object(preserved_copy.s3_key)
     stored_checksums = stored_checksums(preserved_copy)
     replicated_checksum = replicated_checksum(aws_s3_object)
-    return preserved_copy.replicated_copy_not_found! unless aws_s3_object.exists?
+    unless aws_s3_object.exists?
+      preserved_copy.replicated_copy_not_found!
+      Rails.logger.error("Archival Preserved Copy: #{preserved_copy} was not found on #{bucket_name}.")
```

Comment:
Should do your logging off the object *before* you mutate the object.  I.E., change the order of the log and setter lines.